committeeoversightapp/templates/committeeoversightapp/landing_page.html

```diff
@@ -74,6 +43,19 @@ <h3 class="mb-4 text-right">The Senate</h3>
         var url = select.options[select.selectedIndex].value;
         window.location.href=url;
     }
+
+    $(document).ready(function () {
+        $('.committee-table').DataTable({
+          "dom": "Bfrtip",
+          "paging": false,
+          "searching": false,
+          "info": false,
+          "order": [[ 0, "asc" ]],
+          "columnDefs": [
+            { "orderable": false, "targets": [3, 4, 5] },
```

Comment:
Sorting by letter grade yields A, A+, A-, ..., F, F+, F-. Do you want to open an issue to address this later? If it helps, I tweaked the `natural` sort algorithm (https://github.com/datamade/risk-and-reach/pull/155/files#diff-1bf986eed1cc230d18c0075b6800592a) to sort mixtures of numbers and strings in a DataTable. There may be a lighter weight option for this use case!

Reply:
Great point! I'll open an issue to track this. It's possible these letters will just be letters, so we should hold off until Lugar decides.